providers/io/osf/normalizer.py

```diff
@@ -66,7 +66,7 @@ class Project(Parser):
         ctx.embeds.affiliated_institutions.data
     )
     date_updated = tools.ParseDate(ctx.attributes.date_modified)
-    subject = tools.Delegate(Tag, ctx.attributes.category)
+    subject = tools.Delegate(Tag, tools.Maybe(ctx, 'attributes.category'))
```

Comment:
I was trying to avoid creating a tag with `name=''`, which Try does when category is an empty string. It looked like Maybe worked as I wanted, but I guess it just avoided errors by never setting `subject`. How would I set `subject` only when `ctx.attributes.category` is a non-empty string?